src/minio.js

```diff
@@ -1846,10 +1820,13 @@ Client.prototype.selectObjectContent = promisify(Client.prototype.selectObjectCo
 
 // refactored API use promise internally
 Client.prototype.makeBucket = callbackify(Client.prototype.makeBucket)
+Client.prototype.bucketExists = callbackify(Client.prototype.bucketExists)
 Client.prototype.removeBucket = callbackify(Client.prototype.removeBucket)
 Client.prototype.listBuckets = callbackify(Client.prototype.listBuckets)
 
 Client.prototype.statObject = callbackify(Client.prototype.statObject)
+Client.prototype.removeObject = callbackify(Client.prototype.removeObject)
```

Comment:
Looks like this is a duplicate of line 1832, is this intentional?

Reply:
a mistake